index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap" rel="stylesheet" type="text/css">
+    <head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>freeCodeCamp Survey Form</title>
+    
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p> 
+</head>
+
+<body>
+    <div class="background">
+        <div class="layer">
+    <form id="survey-form">
+     <div>
+        <label id="name-label" for="name">Name</label><br>
+        <input id=name type="text" placeholder="Enter your name"> <br>  
+   
+        <label id="email-label" for="email">Email</label><br>
+    <input id="email" type="email" placeholder="Enter your Email"><br>
+   
+    <label id="number-label" for="number">Age (optional)</label><br>
+    <input id="number" type="number" placeholder="Age" min="0" max="100">
+</div>
+<div>
+        <label id="dropdown-label">Which option best describes your current role?</label><br>
+        <select id="dropdown" name="Select an option">
+            <option value="Student">Student</option>
+            <option value="Other">Other</option>
+        </select>
+</div>
+<div>
+    <p>Would you recommend freeCodeCamp to a friend?</p>
+    <input type="radio" id="Rad1" name="recommend" value="Definitely">
+    <label for="Rad1">Definitely</label><br>
+    <input type="radio" id="Rad2" name="recommend" value="Maybe">
+    <label for="Rad2">Maybe</label><br>
+    <input type="radio" id="Rad3" name="recommend" value="Not sure">
+    <label for="Rad3">Not sure</label><br> 
+</div>
+<div>
```

Comment:
Kindly. I recommend you remove the code (DIV) that you do not use

Reply:
Kindly, I suggest you support your comments with why it is good to make them or suggest a solution to the issue.